src/views/LoginPage/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { SocialIcon } from 'react-social-icons';
+import {
+  Container, Row, Col, Form, Alert
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Header from '../../components/header/Header';
+import Button from '../../components/button/Button';
+import { loginUser } from '../../redux/actions/auth';
+import Loader from '../../components/loader/Index';
+import './login.scss';
+
+
+export const Login = ({ history, loginUser: handleLogin, loading }) => {
+  const [userCredentials, setUserCredentials] = useState({
+    email: null,
+    password: null
+  });
+  const [errors, setErrors] = useState([]);
+
+  function updateLocalState(event) {
+    setUserCredentials({ ...userCredentials, [event.target.name]: event.target.value });
+  }
+
+  function renderErrors(errorsArray) {
+    return errorsArray.map(error => <Alert key={error} variant="danger">{error}</Alert>);
+  }
+
+  async function onSubmit(event) {
+    event.preventDefault();
+    const loginRes = await handleLogin(userCredentials);
+    if (!Array.isArray(loginRes)) {
+      return history.push('/');
+    }
+    return setErrors(loginRes);
+  }
+
+  return (
+    <>
```

Comment:
@chingsley if every other element is under the `div`, there's no need for a fragment.

Reply:
You are right. I have taken it out.